kerastuner/engine/base_tuner.py

```diff
@@ -126,9 +127,9 @@ def search(self, *fit_args, **fit_kwargs):
                 # Oracle is calculating, resend request.
                 continue
 
-            self.on_trial_begin(trial)
+            self.on_trial_begin(trial, verbose)
```

Comment:
I think rather than adding arguments to `on_trial_begin` and `on_trial_end`, it might make more sense to do `self._display.verbose = verbose` and then have the display contain the logic for what it should/shouldn't print

That way the logging logic is more self-contained in the display object. WDYT?

Reply:
Sounds good! I just made an update with these changes.